subprojects/gradle-plugin/src/main/java/org/spongepowered/gradle/vanilla/VanillaGradle.java

```diff
@@ -71,6 +73,10 @@
 public final class VanillaGradle implements Plugin<Object> {
 
     private static final String SHADOW_JAR_TASK_CLASS_NAME = "com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar";
+    private static final List<String> SHADOW_PLUGIN_IDS = Arrays.asList(
```

Comment:
so like probably not an issue in practice, but still would you be able to wrap this in a Collections.unmodifiableList? otherwise you can still change entries